packages/frontend/lib/api.ts

```diff
@@ -22,6 +33,9 @@ export const secretAgentApi = ky.extend({
         if (!authInfo) throw new Error('not logged in');
         req.headers.set('sa-admin-auth', authInfo.auth);
         req.headers.set('sa-user-id', authInfo.id);
+        req.headers.set('sa-auth-timestamp', authInfo.timestamp);
+        req.headers.set('sa-auth-nonce', authInfo.nonce);
+        req.headers.set('sa-auth-uri', authInfo.uri);
```

Comment:
do we need checks for any of these on backend?

Reply:
yup it's here https://github.com/dmno-dev/secret-agent/pull/45/files/0dce2fbab4c351a5057924e7cf77c998771f2780#diff-957e0e63aebd31146622624176e572f017b03ef23b0a2d722a6f4b581fa968efR79

Reply:
derp, carry on 😅